packages/jellyfish-api-core/__tests__/category/masternode/setGovHeight.test.ts

```diff
@@ -0,0 +1,82 @@
+import { LoanMasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { createPoolPair, createToken } from '@defichain/testing'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Masternode', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
```

Comment:
We should not access container directly

Reply:
Why so? Probably more than 70% of our test case is doing `container.xxx()`. `Testing` is introduced later for extra shortcuts to speed up test env setup, it's the same under the hood.

Reply:
i could not find my comment from a previous issue. Let me type it here.
@ivan-zynesis yes your argument is valid. we have the `container` before we create the `testing`
But we plan to move `container` creation inside `testing`. That refactor is still not done yet🙁
When we do that, we can reduce the refactor required in other files by not using `container` directly😉

Reply:
Understood. At the moment, this file isn't using any `Testing` lib. Also have to avoid over growing the `Testing` as the testing utilities itself become a huge black box and made things in the main test files become too abstract.

Reply:
yes, there's always a tradeoff if we abstract too much when writing tests. I guess we have to find the middle ground.